cinder-client/src/main/java/com/woorea/openstack/cinder/SchedulerStatsExtension.java

```diff
@@ -0,0 +1,32 @@
+package com.woorea.openstack.cinder;
+
+import com.woorea.openstack.base.client.HttpMethod;
+import com.woorea.openstack.base.client.OpenStackClient;
+import com.woorea.openstack.base.client.OpenStackRequest;
+import com.woorea.openstack.cinder.model.Pools;
+
+/**
+ * Cinder Scheduler Stats Management
+ *
+ * @author Ahmed Saba
+ */
+public class SchedulerStatsExtension {
+
+    private final OpenStackClient CLIENT;
+
+    public SchedulerStatsExtension(OpenStackClient client) {
+        this.CLIENT = client;
+    }
+
+    public List list(boolean detail) {
+        return new List(detail);
+    }
+
+    public class List extends OpenStackRequest<Pools> {
+
+        public List(boolean detail) {
+            super(CLIENT, HttpMethod.GET, (new StringBuilder("/scheduler-stats/get_pools")).append(detail ? "?detail=True":""), null, Pools.class);
```

Comment:
just "detail " returns name of the pools while "detail=True" is required to return extra information like capabilities

Example : detail
{"pools": [{"name": "jk@lvmdriver-1#lvmdriver-1"}, {"name": "jk@CModePhysicaliSCSI#OpenStack_DL386_50G"}, {"name": "jk@CModePhysicaliSCSI#OpenStack_100G_Extra"}, {"name": "jk@CModePhysicaliSCSI#OpenStack_RDO_100G"}]}

With detail=True
{"pools": [{"name": "jk@lvmdriver-1#lvmdriver-1", "capabilities": {"pool_name": "lvmdriver-1", "goodness_function": null, "total_volumes": 6, "provisioned_capacity_gb": 10.0, "timestamp": "2015-04-24T21:49:17.727372", "allocated_capacity_gb": 10, "volume_backend_name": "lvmdriver-1", "thin_provisioning_support": false, "driver_version": "3.0.0", "total_capacity_gb": 10.01, "reserved_percentage": 0, "filter_function": null, "QoS_support": false, "thick_provisioning_support": true, "free_capacity_gb": 0.01, "location_info": "LVMVolumeDriver:jk:stack-volumes-lvmdriver-1:default:0", "max_over_subscription_ratio": 20.0, "vendor_name": "Open Source", "storage_protocol": "iSCSI"}}, {"name": "jk@CModePhysicaliSCSI#OpenStack_DL386_50G", "capabilities": {"pool_name": "OpenStack_DL386_50G", "netapp_thick_provisioned": "false", "timestamp": "2015-04-24T21:49:42.290257", "allocated_capacity_gb": 0, "volume_backend_name": "iSCSIdriver-1", "driver_version": "1.0.0", "total_capacity_gb": 39.58, "reserved_percentage": 0, "netapp_raid_type": "raid_dp", "netapp_disk_type": "SAS", "netapp_nodedup": "true", "netapp_unmirrored": "true", "QoS_support": false, "netapp_nocompression": "true", "free_capacity_gb": 38.73, "netapp_compression": "false", "netapp_mirrored": "false", "netapp_dedup": "false", "netapp_thin_provisioned": "true", "vendor_name": "NetApp", "storage_protocol": "iSCSI"}}, {"name": "jk@CModePhysicaliSCSI#OpenStack_100G_Extra", "capabilities": {"pool_name": "OpenStack_100G_Extra", "netapp_thick_provisioned": "true", "timestamp": "2015-04-24T21:49:42.290257", "allocated_capacity_gb": 0, "volume_backend_name": "iSCSIdriver-1", "driver_version": "1.0.0", "total_capacity_gb": 79.16, "reserved_percentage": 0, "netapp_raid_type": "raid_dp", "netapp_disk_type": "SAS", "netapp_nodedup": "true", "netapp_unmirrored": "true", "QoS_support": false, "netapp_nocompression": "true", "free_capacity_gb": 56.54, "netapp_compression": "false", "netapp_mirrored": "false", "netapp_dedup": "false", "netapp_thin_provisioned": "false", "vendor_name": "NetApp", "storage_protocol": "iSCSI"}}, {"name": "jk@CModePhysicaliSCSI#OpenStack_RDO_100G", "capabilities": {"pool_name": "OpenStack_RDO_100G", "netapp_thick_provisioned": "false", "timestamp": "2015-04-24T21:49:42.290257", "allocated_capacity_gb": 1, "volume_backend_name": "iSCSIdriver-1", "driver_version": "1.0.0", "total_capacity_gb": 79.16, "reserved_percentage": 0, "netapp_raid_type": "raid_dp", "netapp_disk_type": "SAS", "netapp_nodedup": "true", "netapp_unmirrored": "true", "QoS_support": false, "netapp_nocompression": "true", "free_capacity_gb": 57.39, "netapp_compression": "false", "netapp_mirrored": "false", "netapp_dedup": "false", "netapp_thin_provisioned": "true", "vendor_name": "NetApp", "storage_protocol": "iSCSI"}}]}